src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -56,7 +60,42 @@ public void should_map_zip_envelope_correctly() throws Exception {
         assertSameNonScannableItems(dbEnvelope, zipEnvelope);
         assertRightScannableItemsHaveWarnings(dbEnvelope, ocrValidationWarnings);
 
-        // properties specific to DB envelope
+        assertDbEnvelopeSpecificProperties(container, dbEnvelope);
+    }
+
+    @Test
+    public void should_map_document_types_and_subtypes_correctly() throws Exception {
```

Comment:
method says doc type and subtype but you are also checking ocr data ?

Reply:
Have renamed